documentation/modules/auxiliary/admin/http/allegro_rompager_auth_bypass.md

```diff
@@ -0,0 +1,234 @@
+52.  TP-Link  TD-W8901G - V3_140512                                   
+53.  TP-Link  TD-W8901G - V3_100603                                   
+54.  TP-Link  TD-W8901G - V3_100702_TR                                
+55.  TP-Link  TD-W8901G - V3_100901                                   
+56.  TP-Link  TD-W8901G - V6_110119                                   
+57.  TP-Link  TD-W8901G - V6_110915                                   
+58.  TP-Link  TD-W8901G - V6_120418                                   
+59.  TP-Link  TD-W8901G - V6_120213                                   
+60.  TP-Link  TD-W8901GB - V3_100727                                   
+61.  TP-Link  TD-W8901GB - V3_100820                                   
+62.  TP-Link  TD-W8901N - V1_111211                                   
+63.  TP-Link  TD-W8951ND - V1_101124,100723,100728                     
+64.  TP-Link  TD-W8951ND - V1_110907                                   
+65.  TP-Link  TD-W8951ND - V1_111125                                   
+66.  TP-Link  TD-W8951ND - V3.0_110729_FI                              
+67.  TP-Link  TD-W8951ND - V3_110721                                   
+68.  TP-Link  TD-W8951ND - V3_20110729_FI                              
+69.  TP-Link  TD-W8951ND - V4_120511                                   
+70.  TP-Link  TD-W8951ND - V4_120607                                   
+71.  TP-Link  TD-W8951ND - V4_120912_FL                                
+72.  TP-Link  TD-W8961NB - V1_110107                                   
+73.  TP-Link  TD-W8961NB - V1_110519                                   
+74.  TP-Link  TD-W8961NB - V2_120319                                   
+75.  TP-Link  TD-W8961NB - V2_120823                                   
+76.  TP-Link  TD-W8961ND - V1_100722,101122                            
+77.  TP-Link  TD-W8961ND - V1_101022_TR                                
+78.  TP-Link  TD-W8961ND - V1_111125                                   
+79.  TP-Link  TD-W8961ND - V2_120427                                   
+80.  TP-Link  TD-W8961ND - V2_120710_UK                                
+81.  TP-Link  TD-W8961ND - V2_120723_FI                                
+82.  TP-Link  TD-W8961ND - V3_120524,120808                            
+83.  TP-Link  TD-W8961ND - V3_120830                                   
+84.  ZyXEL    P-660R-T3 - 3.40(BOQ.0)C0                        
+85.  ZyXEL    P-660RU-T3 - 3.40(BJR.0)C0
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: ```use auxiliary/admin/http/allegro_rompager_auth_bypass```
+  3. Do: ```set rhost <ip>```
+  4. Do: ```set rport <port>```
+  5. Do: ```set device <device-id>```
+  6. Do: ```run```
+  7. You should be able to login into the device without authentication
+  
+## Scenarios
+
+  Example run against TP-Link TD-8840T with firmware V2_100525:
+```
+msf > use auxiliary/admin/http/allegro_rompager_auth_bypass 
+msf auxiliary(allegro_rompager_auth_bypass) > devices
+
+List of vulnerable devices
```

Comment:
Having this in the docs twice is pretty long and repetitive. I'd vote to remove this (2nd block).
Maybe list the first 3, then do "...clip..." or something similar afterwards.  This way you show its a function, w/o taking up as much space.